test/services/carrier_account.test.js

```diff
@@ -56,6 +57,23 @@ describe('CarrierAccount Service', function () {
     }
   });
 
+  it('creates a UPS carrier account', async function () {
+    const data = {
+      type: 'UpsAccount',
+      account_number: '123456789',
+    };
+
+    try {
+      const carrierAccount = await this.client.CarrierAccount.create(data);
+      await this.client.CarrierAccount.delete(carrierAccount.id);
+      // If we get here, the UPS account was created (and deleted) successfully
+      expect(true).to.equal(true);
```

Comment:
This assertion is meaningless. We should assert the `carrierAccount` attributes like what you have down below for update unit test

Reply:
Resolved